libs/cart/state/src/effects/cart-item.effects.ts

```diff
@@ -103,7 +103,7 @@ export class DaffCartItemEffects<
   @Effect()
   delete$ = this.actions$.pipe(
     ofType(DaffCartItemActionTypes.CartItemDeleteAction),
-    switchMap((action: DaffCartItemDelete<T>) =>
+    concatMap((action: DaffCartItemDelete<T>) =>
```

Comment:
Can you write a test that fails for the `switchMap` but succeeds for this implementation?

Reply:
added

Reply:
also, I made it a mergeMap instead. concatMap holds the request until the previous one is successful, which I don't think we want, especially because the queued apollo service already queues the requests